lib/rubocop/cop/style/frozen_string_literal_comment.rb

```diff
@@ -12,7 +12,7 @@ class FrozenStringLiteralComment < Cop
         include ConfigurableEnforcedStyle
         include FrozenStringLiteral
 
-        MSG = 'Missing frozen string literal comment.'.freeze
+        MSG = 'File must begin with "# frozen_string_literal: true".'.freeze
```

Comment:
I think that's a bit confusing as well. The file should really begin with this - keep in mind there things like shebangs and other magic comments. Also - code snippets in messages should be in backticks, not double quotes.